composer.json

```diff
@@ -24,18 +24,22 @@
   ],
   "minimum-stability": "stable",
   "require": {
+    "php": ">=8.2",
     "guzzlehttp/guzzle": "7.9.3"
   },
   "require-dev": {
-    "phpunit/phpunit": "12.1.4",
+    "phpunit/phpunit": "^11.5.2",
     "symplify/easy-coding-standard": "12.5.16",
-    "testcontainers/testcontainers": "1.0.2"
+    "testcontainers/testcontainers": "1.0.2",
+    "phpstan/phpstan": "^2.1"
   },
   "scripts": {
     "analyze": "vendor/bin/ecs check",
     "format": "vendor/bin/ecs check --fix",
+    "stan": " vendor/bin/phpstan analyze -l 5 src tests",
```

Comment:
Da das ja auch zur Codeanalyse zählt, würde ich vorschlagen, den Befehl in `analyze` mit zu integrieren und die beiden Aufrufe per `&&` zu verketten, um konsistent mit allen anderen Repositories von uns zu bleiben.